packages/react-cosmos-apollo-proxy/src/__tests__/mocking.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```

Comment:
Seems like this test file is more related to rendering the other proxies and adding the ApolloProvider, so maybe rename to `render.js`?

Reply:
Nice one, thanks!